lib/Frontend/CompilerInvocation.cpp

```diff
@@ -406,9 +387,6 @@ static bool ParseLangArgs(LangOptions &Opts, ArgList &Args,
   Opts.DisableAvailabilityChecking |=
       Args.hasArg(OPT_disable_availability_checking);
 
-  if (FrontendOpts.InputKind == InputFileKind::SIL)
```

Comment:
I have just removed this bit which conflicted, let me know if that makes sense @CodaFi